core/src/main/java/nu/yona/server/exceptions/ResourceBasedException.java

```diff
@@ -31,27 +33,17 @@ void onContextStarted(ContextRefreshedEvent event)
 	}
 
 	private static final long serialVersionUID = 8031973645020363969L;
-	/** Holds the parameters for the exception message. */
-	private final Object[] parameters;
-	/** Holds the message id. */
+	private final String[] messageArgs;
 	private final String messageId;
-	/** Holds the HTTP response code to be used. */
 	private final HttpStatus statusCode;
 	private static Translator translator;
 
-	/**
-	 * Constructor.
-	 * 
-	 * @param statusCode The status code of the exception.
-	 * @param messageId The ID of the exception in the resource bundle
-	 * @param parameters The parameters for the message
-	 */
-	protected ResourceBasedException(HttpStatus statusCode, String messageId, Object... parameters)
+	protected ResourceBasedException(HttpStatus statusCode, String messageId, Object... messageArgs)
 	{
 		super(messageId);
 
 		this.messageId = messageId;
-		this.parameters = parameters;
+		this.messageArgs = serializeMessageArgs(messageArgs);

```

Comment:
This I don't like. The message format can in principle use format specifications that depend on the type (see [this](https://docs.oracle.com/javase/8/docs/api/java/text/MessageFormat.html)). By converting everything to string, that won't work anymore. Next, I expect the message format will show "null" when a null value is given, while the new approach would result in an NPE.

Given that this is only done to address a serialization issue that we don't have (as we don't serialize), I propose we revert this part of the change.